example/v3/example.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:postgres/postgres_v3_experimental.dart';
+
+void main() async {
```

Comment:
What do you think: instead of keeping these examples, how hard would it be to turn these into tests (that run on CI and locally)?

Reply:
I agree that tests for this are obviously needed, and started to write some.

But I still think having an example around in addition to tests would be a good way for new users to get a quick overview of the API. Should I remove the remaining example API?